app/views/stories/create.js.erb

```diff
@@ -0,0 +1,15 @@
+<% if @story.errors.any? %>
+
+  $("#new_story_form").html("<%= j render('form', target: [@map_asset, @story], map_asset: @map_asset) %>");
+  rating_binder();
+
+<% else %>
+
+  $('#map_asset #rating_display').empty();
+  $('#map_asset #rating_display').data("score", <%= @story.map_asset.rating %>);
+   asset_rating();
+
+  $("#stories_table").prepend("<%= j render('shared/story_row', story: @story) %>");
+  $(".close-modal").click();
```

Comment:
find a way to close a modal through bootstrap Js functions

Reply:
Done